moxie-protocol/schema.graphql

```diff
@@ -297,6 +309,39 @@ type Portfolio @entity {
   updatedAtBlockInfo: BlockInfo!
 }
 
+type LockInfo @entity {
+  "index"
+  id: ID!
+  "txHash of the transaction"
+  txHash: Bytes!
+  "logIndex of the transaction"
+  logIndex: BigInt!
+  "User's address"
+  user: User!
+  "Spender's address"
+  spender: User!
+  "Subject's token address"
+  subjectToken: SubjectToken!
+  "portfolio"
+  portfolio: Portfolio!
+  "portfolio"
+  spenderPortfolio: Portfolio!
+  "Subject's address"
+  subject: User!
+  "Unlock time in seconds"
+  unlockTimeInSec: BigInt!
+  "Amount locked"
+  amount: BigInt!
+  "isBuy"
+  isBuy: Boolean!
+  "Lock period"
+  lockPeriodInSec: BigInt!
+  "moxie spent to buy the subject token & lock"
+  moxieDepositAmount: BigInt!
+  "created at block information"
+  createdAtBlockInfo: BlockInfo!
```

Comment:
in case of buy & deposit, there should be an order record created for the buy.

Reply:
it's already handled from bondingCurve events